pkgs/development/python-modules/austin-python/default.nix

```diff
@@ -0,0 +1,88 @@
+{ stdenv
+, lib
+, buildPythonPackage
+
+, austin ? null  # Avoid circular dependency between austin / austin-python.
```

Comment:
```suggestion
```

Reply:
(Let's dedup with the thread about $PATH lookup since this only exists as a direct consequence to it.)